gradle.properties

```diff
@@ -1,4 +1,9 @@
 android.enableJetifier=true
+android.defaults.buildfeatures.aidl=false
+android.defaults.buildfeatures.buildconfig=false
+android.defaults.buildfeatures.renderscript=false
+android.defaults.buildfeatures.resvalues=false
+android.defaults.buildfeatures.shaders=false
```

Comment:
Some of them are still enabled by default.

https://developer.android.com/reference/tools/gradle-api/8.0/com/android/build/api/dsl/BuildFeatures